dcos/mesos.py

```diff
@@ -100,6 +100,52 @@ def get_state_summary(self):
         url = self.master_url('master/state-summary')
         return http.get(url).json()
 
+    def slave_file_read(self, slave_id, params):
+        """ See the master_file_read() docs
+
+        :param params: requests `params` argument
+        :type params: dict
+        :returns: files/read.json response
+        :rtype: dict
+        """
+
+        url = self.slave_url(slave_id, 'files/read.json')
+        return http.get(url, params=params).json()
+
+    def master_file_read(self, params):
+        """This endpoint isn't well documented anywhere, so here is the spec
+        derived from the mesos source code:
+
+        request format:
+        {
+            path: absolute path to read
+            offset: start byte location, or -1.  -1 means read no data, and
+                    is used to fetch the size of the file in the response's
+                    'offset' parameter.
+            length: number of bytes to read, or -1.  -1 means read the whole
+                    file.
+        }
```

Comment:
I see. They are documented here. If those params are part of the function signature we could have documented it using pydoc's `:param path:` syntax.